src/sentry/api/endpoints/project_metrics_extraction_rules.py

```diff
@@ -84,8 +84,12 @@ def get(self, request: Request, project: Project) -> Response:
         if not self.has_feature(project.organization, request):
             return Response(status=404)
 
+        query = request.GET.get("query")
```

Comment:
nit: can we find a better name for this, especially since query already has a meaning in the context of the UI?

Reply:
Usually in our API for this type of filtering we use a `query` parameter. I agree that here it can be confusing, and maybe some other parameter name could be better, but we just have to be aware that in that case we are breaking a "convention".
Do you have any suggestion for the parameter name?

Reply:
Ah yeah you're right